src/index.ts

```diff
@@ -1,3 +1,5 @@
+import { Blob } from 'node:buffer';
```

Comment:
This definitely won't work in a browser so this is not a way to go, however `Blob` should be available as a global in basically all browsers at this point and also in nodejs 18

Reply:
I didn't have this defined initially, but building this fails with `Blob` type not-identified. That is why I put this in place. Any other way you'd recommend to build w/o this `import`?

Reply:
:confused: now builds w/o the `import`. I still have the `@types/nodes` though.